src/Components/Event/List/EventList.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+import Tabs from "./Tabs";
+import OngoingEvents from "./OngoingEvents";
+import RecruitingEvents from "./RecruitingEvents";
+import FinishedEvents from "./FinishedEvents";
+import RadioButtons from "./RadioButtons";
+
+export default function App() {
+  const [selectedTab, setSelectedTab] = useState("진행중");
```

Comment:
```suggestion
  const [selectedTab, setSelectedTab] = useState<"진행중"|"모집중"|"종료된 행사">("진행중");
```
이렇게 하면 selectedTab에서 올 수 있는 텍스트를 한정할 수 있어서 에러 방지랑 자동 완성 기능을 쓸 수 있습니다.
추가로 해당 텍스트를
export type Progress ="진행중"|"모집중"|"종료된 행사" ;
타입을따로 선언해서 
useState<Progress>("진행중");
사용하면 나중에 Progress 타입을 export해서 다른부분에도 사용할 수 있을거 같아요

Reply:
넵!!